infra/services/kusto.bicep

```diff
@@ -0,0 +1,57 @@
+targetScope = 'resourceGroup'
+
+@minLength(3)
+@maxLength(50)
+@description('The base resource name.')
+param baseName string = resourceGroup().name
+
+@description('The location of the resource. By default, this is the same as the resource group.')
+param location string = resourceGroup().location
+
+@description('The tenant ID to which the application and resources belong.')
+param tenantId string = '72f988bf-86f1-41af-91ab-2d7cd011db47'
+
+@description('The client OID to grant access to test resources.')
+param testApplicationOid string
+
+
+resource kustoCluster 'Microsoft.Kusto/clusters@2024-04-13' = {
+  name: baseName
+  location: location
+  sku: {
+    name: 'Standard_D12_v2'
```

Comment:
we're hitting SKU issues when deploying to TME:
[BadRequest] The selected SKU is restricted in this region. Please select a different SKU Series.

I was able to create a cluster using "Standard_E2ads_v5"